assets/js/modules/analytics/util/index.js

```diff
@@ -37,21 +37,21 @@ export { calculateOverviewData };
 export { default as parsePropertyID } from './parse-property-id';
 export * from './validation';
 
-export const extractAnalyticsDataForTrafficChart = ( reports ) => {
+export const extractAnalyticsDataForTrafficChart = ( reports, idx = 1 ) => {
```

Comment:
Would be great to add a comment on what `idx` is and why it is here, as it may not be obvious to someone else lacking all that context.

Reply:
Updated.